src/Client.js

```diff
@@ -727,6 +727,7 @@ class Client extends EventEmitter {
                 await this.authStrategy.disconnect();
                 this.emit(Events.DISCONNECTED, 'NAVIGATION');
                 await this.destroy();
```

Comment:
if you run `await this.destroy();` its already close the pupPage so you cant logout

Reply:
I think in case of localAuth, the session folder is deleted with this method execution.